sources/HUBViewControllerImplementation.m

```diff
@@ -963,7 +963,9 @@ - (void)collectionViewCellWillAppear:(HUBComponentCollectionViewCell *)cell
     }
     
     [self componentWrapperWillAppear:wrapper];
-    [self.delegate viewController:self componentWithModel:wrapper.model willAppearInView:cell];
+
+    UIView * const component = HUBComponentLoadViewIfNeeded(cell.component);
```

Comment:
So `nil` check this here first. Also `componentView` or something would be a better name, since it's a `UIView`, not a component 🙂